src/components/avatar/README.md

```diff
@@ -0,0 +1,59 @@
+# Avatar
+A text or image-based graphic to represent a person or user
+
+## When to use
+Use the avatar component to visually represent a person within content or a user interface. This could include:
+
+- A photo avatar of a person who said a quote (such as when used within the `quote` component)
+- A text or image avatar to represent a user who is logged into a system (such as when used within the `identity module` in a `header`)
+
+## When to use something else
+The image avatar should be used with raster images. If a vector graphic is desired, consider using something like a SVG instead.
```

Comment:
I would think about either linking to the definition of _raster image_, or maybe changing up the words a bit. Maybe something like "...used with image formats like .jpg or .png that are sized for use on the web."